### ISSUE TITLE
[Question] Support multi-project setup?

### ISSUE THREAD

**Issue by erni-cadn** [insideapp-oss/sonar-flutter#7]
Will running the analysis within a folder that contains multiple projects work?

e.g. Project structure
```
-src/
    -project_1/
        -test/
        -pubspec.yaml
    -project_2/
        -test/
        -pubspec.yaml
    -project_3/
        -test/
        -pubspec.yaml
    -project_foo/
        -test/
        -pubspec.yaml
```

**Comment by zippy1978** [comment]
Hi @erni-cadn,

Yes, it should work if you provide your multiple sources and test folders into sonar-project.properties, like this :

```
sonar.sources=src/project_1,src/project_2, ...
sonar.exclusions=src/project_1/test/**,src/project_2/test/**,...

sonar.tests=src/project_1/test,src/project_2/test, ...

```

**Comment by arturdm** [comment]
Hi,

thank you for suggestion @zippy1978  - I tried that in my project but analysis fails on `dartanalyzer` sensor with error code 3.

It looks like it passes paths to all the dart files to `dartanalyzer` directly and since the code uses packages it can't find some of dependencies. When I run `dartanalyzer` from the root of the multi project with the exact same paths it fails with the same error code 3 and unresolved symbols message.

I also tried with multi module approach (each module has its own `sonar-project.properties` file) but it fails with the same error because the paths provided are relative to the directory where the sonar analysis is run.

Maybe it would be possible to run dartanalysis for each module separately with paths relative to module. Is it possible without any code changes?
